datasets/mozetl/search_clients_daily/intro.md

```diff
@@ -0,0 +1,42 @@
+`search_clients_daily` is designed to enable client-level search analyses.
+Querying this dataset can be slow;
+consider using `search_aggregates` for coarse analyses.
```

Comment:
suggest:
``` 
However, it is necessary to use `search_clients_daily`
if normalization by client activity
(such as active user count or active time)
is required.
```

Reply:
I think this is covered by the first line.